src/api/spec/models/staging/staging_project_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Staging::StagingProject, vcr: true do
+  let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:staging_workflow) { create(:staging_workflow_with_staging_projects, project: user.home_project) }
+  let(:staging_project) { staging_workflow.staging_projects.first }
+
+  let!(:repository) { create(:repository, architectures: ['x86_64'], project: staging_project, name: 'staging_repository') }
+  let!(:package) { create(:package_with_file, project: staging_project) }
+
+  let!(:status_report) { create(:status_report, checkable: repository) }
+
+  let(:target_project) { create(:project, name: 'target_project') }
+  let(:source_project) { create(:project, name: 'source_project') }
+  let(:target_package) { create(:package, name: 'target_package', project: target_project) }
+  let(:source_package) { create(:package, name: 'source_package', project: source_project) }
+
+  let(:request_attributes) do
+    {
+      target_project: target_project.name,
+      target_package: target_package.name,
+      source_project: source_project.name,
+      source_package: source_package.name
+    }
+  end
+
+  let!(:submit_request) { create(:bs_request_with_submit_action, request_attributes.merge(staging_project: staging_project)) }
+
+  before do
+    allow(Backend::Api::Published).to receive(:build_id).with(staging_project.name, repository.name).and_return('1234')
+    status_report.update(uuid: '1234')
+  end
+
+  describe '#staging_identifier' do
+    let(:staging_project) { create(:staging_project, name: 'openSUSE_41:Staging:myStagingProject') }
```

Comment:
why do you need another `staging_project`? there is one here already: https://github.com/openSUSE/open-build-service/pull/6224/files#diff-5e20b29a349feec5580923e52507b03eR7 :confused:

Reply:
Right, I'll update it. Reason for that is that we have 5 or 6 branches that touch the same code. So every time our feature branch updates, I have to update the code here... Time to get this merged;-)